core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -108,7 +108,8 @@ var ExplorationEditorPage = function() {
   var saveDraftButton = element(by.css('.protractor-test-save-draft-button'));
   var publishExplorationButton = element(
     by.css('.protractor-test-publish-exploration'));
-
+  var selectExplorationLanguage = element(
```

Comment:
Can you specify what kind of element this is? e.g. append button if it is a button.